### PR TITLE
Add utility directory

### PR DESCRIPTION
**Pull request by k2sebeom**
Features include,

1. Add utility directory
2. Refactor topology manager into util
3. Add skeleton code for util

### CODE REVIEW

File: modi/util/that.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            elif button.get_clicked():
+                display.set_text("PONG!!")
+                break
+            time.sleep(0.02)
+        if not running:
+            break
+        time.sleep(1)
+        point = initialize(display, led, speaker, dial)
+        time.sleep(3)
+        display.set_text("Game Over\nScore: " + str(point))
```
Comment (jha929) [inline modi/util/that.py:83]: Use f-string where possible

File: modi/util/that.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            score += 1
+        display.clear()
+    return score
+
+
+def check_complete(bundle):
+    module_names = [type(module).__name__ for module in bundle.modules]
+    expected_names = ["Button", "Dial", "Led", "Speaker", "Display"]
+    is_equal = len(module_names) == len(expected_names)
+    if not is_equal:
```
Comment (jha929) [inline modi/util/that.py:46]: This line and the line above can be merged using "!=".

File: modi/modi.py [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
 import traceback
-from pprint import pprint
 
-from modi.topology_manager import TopologyManager
+from modi.util.topology_manager import TopologyManager
 
 from modi._conn_proc import ConnProc
 from modi._exe_thrd import ExeThrd
-
+from modi.util.that import check_complete
```
Comment (jha929) [inline modi/modi.py:15]: Put this line under the line where the topology manager is being imported.

File: modi/util/that.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                return
+
+    display = bundle.displays[0]
+    button = bundle.buttons[0]
+    led = bundle.leds[0]
+    dial = bundle.dials[0]
+    speaker = bundle.speakers[0]
+    for i in range(5):
+        button.get_pressed()
+        time.sleep(0.02)
```
Comment (jha929) [inline modi/util/that.py:60]: What does this line do?

Comment (k2sebeom) [reply modi/util/that.py:60]: This line checks whether the button is pressed when initializing the MODI object.
The lines below are only executed when the button is pressed at the moment of initialisation.

**Pull request merged**